src/Backups/BackupsWorker.h

```diff
@@ -136,6 +137,7 @@ class BackupsWorker
 
     class ThreadPools;
     std::unique_ptr<ThreadPools> thread_pools;
+    Poco::AutoPtr<Poco::Util::AbstractConfiguration> config;
```

Comment:
@amosbird do I understand correctly that caching `config` will make it non-reloadable? And sicne this `BackupsWorker` is a "global" object it will never be re-loaded?

P.S. this is incredibly useful patch!

Reply:
Yes, and if we’d like to support reloading for `BackupWorker` or something similar, this can be implemented in subsequent PRs. Based on this patch, it should be fairly straightforward to add.

Reply:
>and if we’d like to support reloading for BackupWorker or something similar, this can be implemented in subsequent PRs

Sure thing, I was just thinking maybe it worth to leave a comment in such places